src/main/java/com/uid2/admin/vertx/service/SaltService.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Stream;
```

Comment:
do you want to revert the changes in this file? the before/after seems to be doing exact same thing so might as well keep the original codes?